data/sql/updates/pending_db_world/rev_1692366711663255300.sql

```diff
@@ -0,0 +1,29 @@
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(19, 0, 8356, 0, 0, 47, 0, 8311, 64, 0, 1, 0, 0, 'Prevents quest from being available after completing quest Hallows End Treats for Jespers', NULL);
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 19) AND (`SourceGroup` = 0) AND (`SourceEntry` = 8355) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 47) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 8311) AND (`ConditionValue2` = 64) AND (`ConditionValue3` = 0);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(19, 0, 8355, 0, 0, 47, 0, 8311, 64, 0, 1, 0, 0, 'Prevents quest from being available after completing quest Hallows End Treats for Jespers', NULL);
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 19) AND (`SourceGroup` = 0) AND (`SourceEntry` = 8353) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 47) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 8311) AND (`ConditionValue2` = 64) AND (`ConditionValue3` = 0);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(19, 0, 8353, 0, 0, 47, 0, 8311, 64, 0, 1, 0, 0, 'Prevents quest from being available after completing quest Hallows End Treats for Jespers', NULL);
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 19) AND (`SourceGroup` = 0) AND (`SourceEntry` = 8357) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 47) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 8311) AND (`ConditionValue2` = 64) AND (`ConditionValue3` = 0);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(19, 0, 8357, 0, 0, 47, 0, 8311, 64, 0, 1, 0, 0, 'Prevents quest from being available after completing quest Hallows End Treats for Jespers', NULL);
```

Comment:
This should be able to be done using exclusive groups in quest _addon iirc

Reply:
> This should be able to be done using exclusive groups in quest _addon iirc

Instead, you have to explain to me better what you mean, because I didn't understand, above all I didn't understand what "iirc" is

Reply:
> > above all I didn't understand what "iirc" is
> 
> ["If I remember correctly."](https://www.dictionary.com/browse/iirc#:~:text=abbreviation,correctly%3B%20if%20I%20recall%20correctly.)

I'm Italian, when you use acronyms I can't always understand what you mean, there are acronyms I know and others I don't.
So, when you talk to me if you can speak as clearly as possible, you do me a great favor..

Reply:
@heyitsbench but thanks for the clarification.

Reply:
Sorry. But you still have not addressed this.

If I remember correctly you can fix this with ExclusiveGroups in quest_template_addon. Conditions shouldnt be needed at all

Reply:
Hi @Kitzunu

I've never used this parameter, but reading what it says, I don't think it does the same thing as the conditions I've inserted, the conditions I've inserted cause "The Sub-Quests" to be no longer available after the Main Quest is completed.

![image](https://github.com/azerothcore/azerothcore-wotlk/assets/1884642/a30147f8-6dab-4f86-9724-2740c2fcba00)

Reply:
Reading https://www.azerothcore.org/wiki/quest_template_addon#exclusivegroup I don't think this is the case.

ExclusiveGroup allows to:
- make a quest non-acceptable (or non-visible) until you complete a specific quest or group of quests
- make only one quest completable having a group of quests

in this case we have a group of quest that are not visible when a specific quest is completed, because `They must disappear as soon as Quest (Main) - [Hallow's End Treats for Jesper!](https://www.wowhead.com/quest=8311/hallows-end-treats-for-jesper) is completed.`

If I understood correctly, it's not the case of ExclusiveGroup, btw, we should improve the Keira3 tooltip description or recommend the wiki description which is a bit better of that tooltip

Reply:
Is it not exactly what you just pasted?

> make a quest non-acceptable (or non-visible) until you complete a specific quest or group of quests

ExclusiveGroups can be used to address this

Reply:
> > in this case we have a group of quest that are not visible when a specific quest is completed
> 
> ExclusiveGroups can be used to address this
> 
> > make only one quest completable having a group of quests
> 
> So if you complete quest N, you may not complete X, Y, Z

ExclusiveGroup is a mechanism that allows you to organize different quests into special groups. There are two ways you can use it:

If ExclusiveGroup is a positive number: This lets you create a group of quests where the player can choose only one to complete. For example, if you have three quests numbered 1200, 1201, and 1202, and you want the player to choose only one, you would put the number 1200 in the ExclusiveGroup parameter of all three quests.

If ExclusiveGroup is a negative number: This is about creating a group of quests that all must be completed and rewarded before the next quest can be initiated. For instance, if quest 1000 depends on one of the quests 1200, 1201, and 1202, and all three of these quests have the same negative exclusive group, then all three quests must be completed and rewarded before quest 1000 can begin.

Reply:
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/1884642/e3fef6c8-010d-4d29-bcee-6e81674a2ec1)

Reply:
> If ExclusiveGroup is a positive number: This lets you create a group of quests where the player can choose only one to complete. For example, if you have three quests numbered 1200, 1201, and 1202, and you want the player to choose only one, you would put the number 1200 in the ExclusiveGroup parameter of all three quests.

This, precisely